release-notes.md

```diff
@@ -1,5 +1,14 @@
 # Release notes
 
+## June 27, 2019
+
+The following updates were made in this release:
+
+**iOS Target 2.1.1 & Android Target 1.1.1**
+
+- Use `target.propertyToken` configuration setting to configure at_property_token generated from Target UI, instead of passing as mbox parameter.
```

Comment:
Please reword so it reads as follows:

"Use the `target.propertyToken` configuration setting to configure the `at_property_token` that is generated from the Target UI, instead of passing the token as an mbox parameter."